internal/mempool/mempool.go

```diff
@@ -276,7 +277,7 @@ func (txmp *TxMempool) CheckTx(
 	res, err := txmp.proxyAppConn.CheckTx(ctx, &abci.RequestCheckTx{Tx: tx})
 	if err != nil {
 		txmp.cache.Remove(tx)
-		res.Log = err.Error()
+		res.Log = txmp.AppendCheckTxErr(res.Log, err.Error())
```

Comment:
Whats the motivation for this change? Just wondering if it'll break any integration logic. IRRC we had issues with changing how we serialized events?

Reply:
this change is to surface the checkTx error log to cosmos level so that it can correctly be parsed: https://github.com/sei-protocol/sei-cosmos/pull/258
this shouldn't affect any other checkTx flows and in the worst case the res.Log will be empty (used to be by default empty in ABCI++)

Reply:
i commented in the other PR, but can we append a new field rather than replace?